<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="View">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>0.000000000000000</width>
      <height>0.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>23</height>
     </size>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>80</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>150</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="score">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>60</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pause">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>210</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="min">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="sec">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>80</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Playmusic">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>180</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Play music</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="Bats">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="People">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>120</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="up4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade Range</string>
    </property>
   </widget>
   <widget class="QPushButton" name="up5">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Upgrade Speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd4">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd5">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>50</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="fps">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>220</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
